medusa/__main__.py

```diff
@@ -777,6 +777,9 @@ def initialize(self, console_logging=True):
             app.TRAKT_SYNC = bool(check_setting_int(app.CFG, 'Trakt', 'trakt_sync', 0))
             app.TRAKT_SYNC_REMOVE = bool(check_setting_int(app.CFG, 'Trakt', 'trakt_sync_remove', 0))
             app.TRAKT_DEFAULT_INDEXER = check_setting_int(app.CFG, 'Trakt', 'trakt_default_indexer', 1)
+            if app.TRAKT_DEFAULT_INDEXER == 3:
```

Comment:
Dont worry. Trakt dev will let me know when tvmaze is added. Then I will remove this